[2025-01-01 15:19:02,948 E 47586 47586] (raylet) logging.cc:97: Unhandled exception: N5boost10wrapexceptINS_6system12system_errorEEE. what(): Invalid argument
[2025-01-01 15:19:02,950 E 47586 47653] (raylet) logging.cc:97: Unhandled exception: N5boost10wrapexceptINS_6system12system_errorEEE. what(): Invalid argument
[2025-01-01 15:19:03,006 E 47586 47653] (raylet) logging.cc:104: Stack trace: 
 /home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0x51a7fa) [0x5588ba3027fa] ray::operator<<()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0x51cfa8) [0x5588ba304fa8] ray::TerminateHandler()
/lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa37c) [0x7fa4dc8f237c]
/lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3e7) [0x7fa4dc8f23e7]
/lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa699) [0x7fa4dc8f2699]
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0x15bf3e) [0x5588b9f43f3e] boost::throw_exception<>()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0xa77360) [0x5588ba85f360] boost::asio::detail::do_throw_error()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0xa77560) [0x5588ba85f560] boost::asio::ip::make_address()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0xa77585) [0x5588ba85f585] boost::asio::ip::make_address()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0x51761c) [0x5588ba2ff61c] ParseUrlEndpoint()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0x3347e6) [0x5588ba11c7e6] plasma::PlasmaStore::PlasmaStore()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0x32d668) [0x5588ba115668] plasma::PlasmaStoreRunner::Start()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0x2c92c5) [0x5588ba0b12c5] std::thread::_State_impl<>::_M_run()
/home/jh146/ptl_rdf/QFL-MLNCP-NeurIPS/fedqnn-3/lib/python3.10/site-packages/ray/core/src/ray/raylet/raylet(+0xac4010) [0x5588ba8ac010] execute_native_thread_routine
/lib/x86_64-linux-gnu/libpthread.so.0(+0x8609) [0x7fa4dcb8b609] start_thread
/lib/x86_64-linux-gnu/libc.so.6(clone+0x43) [0x7fa4dc75a353] __clone

*** SIGABRT received at time=1735766343 on cpu 7 ***
PC: @     0x7fa4dc67e00b  (unknown)  raise
    @     0x7fa4dcb97420       2208  (unknown)
    @     0x7fa4dc8f237c  (unknown)  (unknown)
    @     0x7fa4dc8f25c0  (unknown)  (unknown)
[2025-01-01 15:19:03,006 E 47586 47653] (raylet) logging.cc:361: *** SIGABRT received at time=1735766343 on cpu 7 ***
[2025-01-01 15:19:03,006 E 47586 47653] (raylet) logging.cc:361: PC: @     0x7fa4dc67e00b  (unknown)  raise
[2025-01-01 15:19:03,006 E 47586 47653] (raylet) logging.cc:361:     @     0x7fa4dcb97420       2208  (unknown)
[2025-01-01 15:19:03,006 E 47586 47653] (raylet) logging.cc:361:     @     0x7fa4dc8f237c  (unknown)  (unknown)
[2025-01-01 15:19:03,006 E 47586 47653] (raylet) logging.cc:361:     @     0x7fa4dc8f25c0  (unknown)  (unknown)
